canvas_account_admin_tools/static/canvas_account_admin_tools/css/dashboard.css

```diff
@@ -1,8 +1,151 @@
+
+a {
+    text-decoration: none;
+}
+a:focus {
+    text-decoration: none;
+}
+a:hover {
+    text-decoration: none;
+}
+h1 {
+    font-size: 1.8em;
+    font-weight: 300;
+    line-height: 1.5;
+}
+h2 {
+    display: block;
+    font-size: 1.5em;
+    font-weight: bold;
+    -webkit-margin-before: 0.83em;
+    -webkit-margin-after: 0.83em;
+    -webkit-margin-start: 0px;
+    -webkit-margin-end: 0px;
+}
+p {
+    color: gray;
+    font-size: 14px;
+    line-height: 1;
+    margin: 0;
+    padding-top: 1em;
+}
+
+/* custom classes */
+
+.card {
+    box-sizing: border-box;
+    box-shadow: 0 0 10px rgba(0,0,0,0.2);
+    border-bottom: 4px solid #eee;
+    background: #fff;
+    cursor: pointer;
+    display: inline-block;
+    font-family: "Helvetica Neue",Helvetica,Arial,sans-serif;
+    margin: 0 36px 36px 0;
+    vertical-align: top;
+    width: 262px;
+}
+
+.card-body {
+    box-sizing: border-box;
+    position: relative;
+}
+.card-content {
+    box-sizing: border-box;
+    padding: 18px 18px 0;
+}
+
+.card-content_alert {
+    background: #ededed;
+    height: 12.5em;
+}
+.card_color {
```

Comment:
Can this be either `.card_color1` (see comment about `.card-header_color5`), or, since we don't use different colors in this interface, it can be combined with `.card`?